core/pkg/service/flag-evaluation/connect_service.go

```diff
@@ -119,7 +120,14 @@ func (s *ConnectService) setupServer(svcConf service.Configuration) (net.Listene
 		s.eventingConfiguration,
 		s.metrics,
 	)
-	path, handler := schemaConnectV1.NewServiceHandler(fes, svcConf.Options...)
+
+	marshalOpts := WithJSON(
+		// json parsing configuration - we emit "unpopulated" fields (falsy fields are not dropped)
+		protojson.MarshalOptions{EmitUnpopulated: true},
+		protojson.UnmarshalOptions{DiscardUnknown: true},
```

Comment:
Does this change the unmarshaling behavior from what we had?

Reply:
Only in the way we want. `DiscardUnknown` was the previous behavior, see [here](https://github.com/bufbuild/connect-go/blob/main/codec.go#L176).